### ISSUE TITLE
don't overwrite existing files - use them or backup them

### ISSUE THREAD

**Issue by c33s** [docwhat/itsalltext#25]
yesterday i wrote quite  a long text. today i wanted to finish the writing but pressing edit again, truncated the existing file to zero. 

my editor was configured to automatically reload changed files. having my editor open on the 2nd screen, i watched my text disappearing.

the request is similar to https://github.com/docwhat/itsalltext/issues/24

please add some kind of safety machanism for such a thing, its . if pressing edit and its all text notices a file with this hash allready exists, it should use the existing one (i can easily ctrl+a, ctrl+x to clean everything but i can't press ctrl+... to get my text back) or at least move the existing one to .bak (or something like that).

edit:
https://github.com/docwhat/itsalltext/issues/14

> http://docwhat.org/files/iat/revisit.html?
> Right-Click on the edit button. Choose the option “Edit using previous '.txt' file”
> The phrase "visit 1" should be in your editor. 

i have no  “Edit using previous '.txt' file” menu entry but the behavior of using previous should be default or at least configurable

**Comment by baby-gnu** [comment]
I agree with @docwhat and I want to explain a little the `Ask the user` case.

I'm using ITA with [Emacs server/client](https://www.gnu.org/software/emacs/manual/html_node/emacs/Emacs-Server.html#Emacs-Server), when I edit wikipage on some poor wiki software I can have some conflicts, in this case I want:

* Keep my Emacs buffer open to keep **my** text
* Refresh the Firefox page to get the new text written by others
* Edit the page with ITA to get the new text in a new Emacs buffer
* Use [Ediff](https://www.gnu.org/software/emacs/manual/html_mono/ediff.html) to resolve the conflict
* Save the merged text.

This requires to create the file under a new name, so the plugin may ask to overrite the file or create a new one (maybe using a timestamp name).

Regards.

**Comment by xOneca** [comment]
¿Why do you let your editor to destroy your work by automatically reloading the file?

In my case it always asks me what to do, and if the file is modified externally, I can overwrite with *my* version, or save to another location so I have both versions available.
